src/frequenz/sdk/actor/channel_registry.py

```diff
@@ -7,25 +7,23 @@
 MIT
 """
 
-from typing import Any, Dict, Generic, TypeVar
+from typing import Any, Dict
 
 from frequenz.channels import Broadcast, Receiver, Sender
 
-T = TypeVar("T")
 
-
-class ChannelRegistry(Generic[T]):
+class ChannelRegistry:
```

Comment:
Because Luca pointed out in the other issue that we probably have multiple key types that will need to share a single registry.

Also I realized that generic key types will work only as long as we are in the single process space.  And we will have to switch to strings if we make it multi-process.

So you were right earlier :-)